### PR TITLE
AWS Target doc updates

### PR DESCRIPTION
**Pull request by cab105**
Remove the `aws.md` target document as it reflects the internal target installation, and replaced it with the per-CRD target documentation.

One thing to note is that the s3 target, while it exists in the target repo, does not have a frontend component yet (see https://github.com/triggermesh/frontend/issues/1472).

The AWS Sources will be a part of a second PR.

### CODE REVIEW

File: docs/targets/awskinesis.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with the AWS Kinesis Target was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
+
+For more information about using AWS Kinesis, please refer to the [AWS documentation][docs].
+
+## Event Types
+
+The AWS Kinesis Target leaves the [CloudEvent][ce] type definition to the discretion of
+the implementer given the nature flexible nature of Kinesis.
```
Comment (sebgoa) [inline docs/targets/awskinesis.md:48]: nature...nature

File: docs/targets/awslambda.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with the AWS Lambda Target was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
+
+For more information about using AWS Lambdas, please refer to the [AWS documentation][docs].
+
+## Event Types
+
+The AWS Lambda Target leaves the [CloudEvent][ce] type definition to the discretion of
+the implementer given the nature flexible nature of AWS Lambdas.
```
Comment (sebgoa) [inline docs/targets/awslambda.md:45]: same

File: docs/targets/awssns.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Event Target for AWS SNS
+
+This event Target receives [CloudEvents][ce] over HTTP and invokes an AWS
+Simple Notification Service (SNS).
```
Comment (sebgoa) [inline docs/targets/awssns.md:4]: an AWS SNS endpoint ?

File: docs/targets/awssns.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with the AWS SNS Target was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
+
+For more information about using AWS Simple Notification Service, please refer to the [AWS documentation][docs].
+
+## Event Types
+
+The AWS SNS Target leaves the [CloudEvent][ce] type definition to the discretion of
+the implementer given the nature flexible nature of AWS SNS.
```
Comment (sebgoa) [inline docs/targets/awssns.md:45]: same

File: docs/targets/awssqs.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with the AWS SQS Target was successfully created.
+
+![Bridge status](../images/bridge-status-green.png)
+
+For more information about using AWS Simple Queue Service, please refer to the [AWS documentation][docs].
+
+## Event Types
+
+The AWS SQS Target leaves the [CloudEvent][ce] type definition to the discretion of
+the implementer given the nature flexible nature of AWS SQS.
```
Comment (sebgoa) [inline docs/targets/awssqs.md:45]: same

**Review by sebgoa — APPROVED** [review]

File: docs/targets/awskinesis.md (hunk 1 above)
Comment (cab105) [reply docs/targets/awskinesis.md:48]: image

**Pull request merged**